style/select.css

```diff
@@ -0,0 +1,362 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);.Select {
```

Comment:
if we are using cimpress' select component - why exactly do we need this stylesheet?

Reply:
what am I missing here?

Reply:
@rnowosielski Yes, I can include it instead.

@istanishev For reasons, Cimpress UX provides external style sheets for certain open source components. My reading is that they have to be included in the code for the component to match the Cimpress style guide. `react-select` is one such component.

Vide https://static.ux.cimpress.io/react-components/latest/:

> Some complex components have not yet been implemented in @cimpress/react-components. Instead, recommended open-source components have been been skinned to be style-guide compliant. Find them [https://static.ux.cimpress.io/os-component-skins/latest/index.html](here).

Indeed, when I used the component without the Cimpress style sheet, it looked super off.

I think that the concern about overwriting a CSS namespace can be dismissed, given that the stylesheet comes from Cimpress UX.

Reply:
Well, if we hard code and there is a new version published by Cimpress UX guys, there will be a clash ;) But that's not going to happen often (I hope). If we can include a link, that would be perfect.